libc-test/build.rs

```diff
@@ -289,6 +298,24 @@ fn main() {
             "FILE_ATTRIBUTE_INTEGRITY_STREAM" |
             "ERROR_NOTHING_TO_TERMINATE" if mingw => true,
 
+            // these are not in musl
+            "SOL_NETLINK" |
+            "SOL_UDP" |
+            "SOL_AX25" |
+            "SOL_ATALK" |
+            "SOL_NETROM" |
+            "SOL_IPX" |
+            "SOL_ROSE" if musl => true,
```

Comment:
Bionic seems to be the only libc implementation that takes a reasonable approach (automating) to keeping constants like this from the kernel reasonably up-to-date (it would be nice if this crate could move toward a similar approach for appropriate items).

These constants get passed straight to the kernel and I do not see any potential conflict with musl.  The kernel takes modification of constants like this very seriously, so I believe the risk of inclusion is extremely low.  Without definitions like this in libc, they must be defined by downstream crates (possibly inconsistently).

I don't think that the failure of musl/glibc to provide a complete set of constants to the operating system should prevent us from doing so.